log2ram.conf

```diff
@@ -14,12 +14,12 @@ USE_RSYNC=false
 
 # If there are some errors with available RAM space, a system mail will be send
 # Change it to false and you will have only a log if there is no place on RAM anymore.
-MAIL=true
+MAIL=false
```

Comment:
Don't change the default

Reply:
The mail option for default needs to go its quite strange to expect a mail install such as postfix on limited hardware that would employ L2R.
But as said later I am asking about writes and the whole sync to disk routine that likely creates as much block wear as logging to flash.
Its dependent on frequency of logging  but with the majority of expected applications it highly likely L2R is currently providing little benefit.
It writes out every out complete files for no reason and it doesn't matter about conf settings apart they are the conf settings I am asking you to give a go?